### ISSUE TITLE
Quote escaping

### ISSUE THREAD

**Issue by tamboril** [davidledwards/zookeeper#23]
Hi.  Love the CLI.  But I can't seem to come up with a way to have double quotes in a value when
setting with (e.g.):

     zk -c 'set -f /params "params:\"value: {key: 10}\""' localhost

...except for putting the value into a file and using `@file`.  Maybe if I could say `@stdin` and pipe my value into it, that would do.

**Comment by davidledwards** [comment]
Looks like I need to fix this problem. In the meantime, try the following:

`echo "..." | zk localhost`

**Comment by tamboril** [comment]
That's cool that you can do that, but it doesn't help.  I still can't set a value containing a double quote.

**Comment by davidledwards** [comment]
Ah yes, makes sense. I will get something out soon.

**Comment by davidledwards** [comment]
Finally got around to fixing this issue. A new section in the README explains how to format quoted strings.

https://github.com/davidledwards/zookeeper/tree/develop-1.4.1/zookeeper-cli#working-with-quoted-strings

**Comment by davidledwards** [comment]
Fixed in [1.4.1](https://github.com/davidledwards/zookeeper/tree/release-1.4.1).

The example provided in the issue should now parse correctly.

```shell
zk -c 'set -f /params "params:\"value: {key: 10}\""' localhost
```
